vscode-trace-extension/src/utils/backend-tsp-client-provider.ts

```diff
@@ -41,7 +41,23 @@ export const addTspClientChangeListener = (listenerFunction: (tspClient: TspClie
 export async function isUp(): Promise<boolean> {
     const health = await getTspClient().checkHealth();
     const status = health.getModel()?.status;
-    return health.isOk() && status === 'UP';
+    const serverIsUp = health.isOk() && status === 'UP';
+    vscode.commands.executeCommand('setContext', 'traceViewer.serverOn', serverIsUp);
```

Comment:
Update the JSDoc to mention that the modified context?

Reply:
addressed